flight/PiOS/Common/pios_hal.c

```diff
@@ -486,6 +507,8 @@ void PIOS_HAL_ConfigurePort(HwSharedPortTypesOptions port_type,
 #if defined(PIOS_INCLUDE_FRSKY_SENSOR_HUB)
 			PIOS_HAL_ConfigureCom(usart_port_cfg, 0, PIOS_COM_FRSKYSENSORHUB_TX_BUF_LEN, com_driver, &port_driver_id);
 			target = &pios_com_frsky_sensor_hub_id;
+			if (sbus_toggle)
```

Comment:
Is this appropriate?  The code I moved from other targets didn't have this-- and since you need an output inverter too I'm not sure why this is here.